packages/tsconfig.settings.json

```diff
@@ -6,7 +6,7 @@
         "sourceMap": true,
 
         "module": "commonjs",
-        "target": "es5",
+        "target": "es6",
```

Comment:
is there a reason why this was `es5`?

Reply:
Because Studio previously couldn't bundle ES6 dependencies as part of its build since it was CRA 1.0. Now that it's 2.0, this is fine.